test/functional/repository/create_test.rb

```diff
@@ -0,0 +1,41 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+
+describe "create repository" do
+  let(:org_id) { 1 }
+  let(:product_id) { 2 }
+  let(:name) { "repo1" }
+  let(:content_type) { "yum" }
+
+  it 'with basic options' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
+        'product_id' => product_id,
+        'content_type' => content_type,
+        'name' => name)
+    end
+
+    command = %W(repository create --organization-id #{org_id} --product-id #{product_id}
+                 --content-type #{content_type} --name #{name})
+
+    assert_equal(0, run_cmd(command).exit_code)
+  end
+
+  it 'with ssl options by name' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
```

Comment:
Instead of passing a block here, it's much easier to debug failed tests if we call `with_params` directly:

```rb
api_expects(:repository, :create, 'create a repository').with_params(...)
```